packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -3,19 +3,29 @@ import 'dart:html' as html;
 
 import 'package:flutter/services.dart';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'package:google_sign_in_web/src/gapi.dart';
```

Comment:
nit: prefer to use relative imports for same-package imports (i.e. `import 'src/gapi.dart';`)

Reply:
I'll change this, but I don't know enough dart to have been bitten by any issue with package vs relative imports. Any reason for this? 

I did this because I think I remember reading somewhere in the dart docs something along the lines "always prefer import from package:blah, because it always works" or similar :/